### ISSUE TITLE
newlines_after_classes false positive with newline after guard clause

### ISSUE THREAD

**Issue by jaredmoody** [coffeelint/coffeelint#28]
```
class TestClass
  constructor: ->
    @locked = true
    @count = 0

  increment: ->
    return if @locked

    count += 1
```

```
> yarn run coffeelint test.coffee                                                                                                                        ruby-2.6.5
yarn run v1.21.1
$ /Users/jaredmoody/code/coffeelint_test/node_modules/.bin/coffeelint test.coffee
  ✗ test.coffee
     ✗ #8: Wrong count of newlines between a class and other code. Expected 1 got 2.

✗ Lint! » 1 error and 0 warnings in 1 file
```

**Comment by UziTech** [comment]
PRs are welcome 😁

**Comment by UziTech** [comment]
Is this still an issue in v3.1.2?

**Comment by jaredmoody** [comment]
Yes, this is still present in 3.1.2

**Comment by UziTech** [comment]
Can you provide your config file?

**Comment by jaredmoody** [comment]
I built an [example repo](
https://github.com/jaredmoody/coffeelint-newlines-after-classes-example) to demonstrate -  I think depending on how you interpret the rule, this could be correct (in 3.1.2 at least).

https://github.com/jaredmoody/coffeelint-newlines-after-classes-example/edit/master/example.coffee

This sure seems like one newline after the class to me, but if you consider the newline at the end of line 9 to be "after" the class - then 2 is correct.

**Comment by jaredmoody** [comment]
Update: Nevermind, confirmed this is still an issue and added a second file to the repo showing the problem - in [example2.coffee](https://github.com/jaredmoody/coffeelint-newlines-after-classes-example/blob/master/example2.coffee) there's not guard clause so it gets interpreted as 1 newline - in example.coffee it gets interpreted as two - so there's no configuration that will work for both of these files. I would have to add an additional newline after the class in `example2.coffee` to get it to pass.

**Comment by UziTech** [comment]
Is the repo private? It gives me a 404

**Comment by jaredmoody** [comment]
Yes, sorry, now public.

**Comment by UziTech** [comment]
I created PR #52 to fix this. I'm not sure why but this rule was counting the number of blank lines before the last line of the class as well as after.
